src/samplers/sampler.jl

```diff
@@ -83,7 +83,7 @@ assume{T<:Distribution}(spl::Void, dists::Vector{T}, vn::VarName, var::Any, vi::
     end
   end
 
-  acclogp!(vi, sum(logpdf(dist, rs, istrans(vi, vns[1]))))
+  acclogp!(vi, sum(logpdf_with_trans(dist, rs, istrans(vi, vns[1]))))
```

Comment:
@yebai Plz have a look at this change - do you agree what we had before was a bug?

Reply:
@yebai This is actually (mathematically) correct ... as there is another bug - we didn't use customised `init` here for vectorised assume at all. I will push another commit to fix it.

Reply:
yes - it looks correct to me.